design/src/components/Glogg/Glogg.jsx

```diff
@@ -0,0 +1,59 @@
+import React from 'react';
+import Champagne from "/Users/durreshewarmalik/repos/HTML_CSS/GV_Design-Sprint/design/src/assets/img/champagne.jpg";
+import Pear from "/Users/durreshewarmalik/repos/HTML_CSS/GV_Design-Sprint/design/src/assets/img/berries.jpg";
+import Berries from "/Users/durreshewarmalik/repos/HTML_CSS/GV_Design-Sprint/design/src/assets/img/margaret.jpg";
+import Saffron from "/Users/durreshewarmalik/repos/HTML_CSS/GV_Design-Sprint/design/src/assets/img/saffron.jpg";
+
+
+
+
+
+const Glogg = () => {
+    return (
+        <div className="glogg-container">
+            <div className="glogg-row1">
+                <h1 className="glogg-h1">Årets glöggsmaker</h1>
```

Comment:
Is there a reason for this specific class, `glogg-h1`? Or would `.glogg-row1 h1` been sufficient?

Reply:
det räcker, det är bara jag som är extra med specifika namn